gef-ui/server/controllers/submit-to-ukef/index.js

```diff
@@ -0,0 +1,71 @@
+import { PROGRESS } from '../../../constants/index';
+import * as api from '../../services/api';
+import { validationErrorHandler } from '../../utils/helpers';
+
+const MAX_COMMENT_LENGTH = 400;
+
+const submitToUkef = async (req, res) => {
+  const { params } = req;
+  const { applicationId } = params;
+  try {
+    return res.render('partials/submit-to-ukef.njk', {
+      applicationId,
+      maxCommentLength: MAX_COMMENT_LENGTH,
+    });
+  } catch (err) {
+    return res.render('partials/problem-with-service.njk');
+  }
+};
+
+const createSubmissionToUkef = async (req, res, next) => {
+  const { params, body } = req;
+  const { userToken } = req.session;
+  const { applicationId } = params;
+  const { comment } = body;
+  const application = await api.getApplication(applicationId);
+  const checker = await api.getUserDetails(application.userId, userToken);
+
+  // TODO : Add some validation here to make sure that the whole application is valid
+  //    Cross check things like user allowed to access, bank it correct, etc
+  //    If it is in an appropriate status to be submitted.
+  //    Consider using an application model so that we have centralised place for reuse.
```

Comment:
Do we have a jira ticket for this, so that it's not forgotten

Reply:
We've got a meeting tomorrow morning that covers this - it's related to the ticket that Paul is working on.

Reply:
ok cool - need to do this on the API side as well, i've added todo comment